common-test/src/main/resources/connector/discovery/acme-metadata.json

```diff
@@ -1,10 +0,0 @@
-{
```

Comment:
none of the following json files were used (they exist and were used in the APF project's tests but not here, so it's probably just copy-paste leftovers)